### PR TITLE
Added an increasing chance for malfunction on repeated failed polling for MM helmets

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

When a poll for a mind-magnified monkey expires without anyone taking the roll, the helmet's malfunction chance will rise by seven. By default, it's '-7', allowing for two safe uses, as explained in the updated description.

After two uses, there's a 7 + n chance of a malfunction, which can be the monkey turning hyper-aggressive, becoming a hostile gorilla, exploding into a shower of gibs, or dying on the spot.

After two uses, it sparks and emits smoke for two minutes. After a third, a glass break sound plays and the smoke turns worse. After the fourth it buzzes and grey smoke emanates. It's very clear when you're being told to stop!
## Why It's Good For The Game

Sometimes a subset of annoying people like to throw the helmet onto monkeys over and over and over until someone caves in. I know you can mute these notifications, but that blocks them for the rest of the round, and it's a bit of a stinker that they can just spam deadchat continously.

The eventual malfunction is telegraphed in advance and is still pretty funny. The experimental helmet that makes monkeys sentients, should, I think, have some risk when using it, especially when *ab*using it.
## Changelog
:cl:
add: Added an increasing chance for malfunction on repeated failed polling for MM helmets
/:cl:

### CODE REVIEW

**Comment by Loafin34** [comment]
you can say "never for this round" on MM helmets

**Comment by carlarctg** [comment]
ok

File: code/modules/clothing/head/mind_monkey_helmet.dm [hunk 1]
```diff
@@ -55,9 +60,40 @@
...
 		magnification = null
 		visible_message(span_notice("[src] falls silent and drops on the floor. Maybe you should try again later?"))
-		playsound(src, 'sound/machines/buzz-sigh.ogg', 30, TRUE)
+		var/particle_path
+		switch(rage_chance)
+			if(-7 to 0)
+				user.visible_message(span_notice("[src] falls silent and drops on the floor. Try again later?"))
+				playsound(src, 'sound/machines/buzz-sigh.ogg', 30, TRUE)
+				particle_path = null
+			if(7 to 13)
```
Comment (SuperNovaa41) [inline code/modules/clothing/head/mind_monkey_helmet.dm:69]: all looks good just change these numbers to defines :+1:

**Review by SuperNovaa41 — CHANGES_REQUESTED** [review]

File: code/modules/clothing/head/mind_monkey_helmet.dm (hunk 1 above)
Comment (carlarctg) [reply code/modules/clothing/head/mind_monkey_helmet.dm:69]: sure klet me just make it more confusing by turning the incrementing the easily readable 7% probability into random arbitrary defines that cant be understood intuitively at a glance

Comment (SuperNovaa41) [reply code/modules/clothing/head/mind_monkey_helmet.dm:69]: make them easy to understand at a glance then

**Pull request merged**